service/src/main/java/tds/assessment/repositories/impl/AssessmentQueryRepositoryImpl.java

```diff
@@ -38,9 +38,13 @@ public AssessmentQueryRepositoryImpl(final DataSource dataSource) {
                 "selectionalgorithm AS selectionAlgorithm, \n" +
                 "startAbility, \n" +
                 "S.name AS subject, \n" +
-                "A.virtualtest AS assessmentKey\n" +
+                "A.virtualtest AS assessmentKey, \n" +
+                "P.propname, \n" +
+                "P.propvalue, \n" +
+                "P.propdescription \n" +
             "FROM itembank.tblsetofadminsubjects A \n" +
             "LEFT JOIN itembank.tblsubject S ON S._key = A._fk_Subject \n" +
+            "LEFT JOIN itembank.tblitemprops P ON P.isactive = 1 and propname = 'Language' AND P._fk_AdminSubject = A._key \n" +
```

Comment:
The left join is for segmented assessments.  The base assessment doesn't have props but instead the individual segments do however we also want that base assessement record.